xbmc/platform/posix/utils/PosixInterfaceForCLog.cpp

```diff
@@ -83,21 +83,24 @@ void CPosixInterfaceForCLog::PrintDebugString(const std::string &debugString)
 #endif // _DEBUG
 }
 
-void CPosixInterfaceForCLog::GetCurrentLocalTime(int &hour, int &minute, int &second, double &milliseconds)
+void CPosixInterfaceForCLog::GetCurrentLocalTime(int& year, int& month, int& day, int &hour, int &minute, int &second, double &milliseconds)
```

Comment:
Oh screw this, I saw just now that the `&` were already inconsistent between header and cpp file before... Can you change it to the way you had it before just for this occurrence please? Sorry for the trouble